src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5082,10 +5082,10 @@ def create_functionapp(cmd, resource_group_name, name, storage_account, plan=Non
         # validate cpu and memory parameters.
         _validate_cpu_momory_functionapp(cpu, memory)
 
-        if (workload_profile_name is not None):
+        if workload_profile_name is not None:
             functionapp_def.workload_profile_name = workload_profile_name
 
-        if (cpu is not None and memory is not None):
+        if cpu is not None and memory is not None:
```

Comment:
Pylint does not find this error, reported to https://github.com/pylint-dev/pylint/issues/10084